templates/stamp/utilities.sh

```diff
@@ -755,6 +760,27 @@ EOF
     sed -i "s/{SMTP_AUTH_USER}/${SMTP_AUTH_USER}/I" $SMTP_CONFIG_FILE
     sed -i "s/{SMTP_AUTH_USER_PASSWORD}/${SMTP_AUTH_USER_PASSWORD}/I" $SMTP_CONFIG_FILE
 
+    # Configure ALIASES
+    ALIAS_CONFIG_FILE="/etc/ssmtp/revaliases"
+
+    if [[ -f $ALIAS_CONFIG_FILE ]]; then
+        log "Removing existing ALIAS configuration"
+        rm $ALIAS_CONFIG_FILE
+    fi
+
+    log "Creating new ALIAS configuration file"
+    tee $ALIAS_CONFIG_FILE > /dev/null <<EOF
+root:{CLUSTER_ADMIN_EMAIL}:{SMTP_SERVER}:{SMTP_SERVER_PORT}
+postmaster:{CLUSTER_ADMIN_EMAIL}:{SMTP_SERVER}:{SMTP_SERVER_PORT}
+{OSADMIN_USERNAME}:{CLUSTER_ADMIN_EMAIL}:{SMTP_SERVER}:{SMTP_SERVER_PORT}
```

Comment:
One is the place holder and other other is the variable. Not a typo but it's not a bad idea to align the two